_data/apps.yml

```diff
@@ -1298,3 +1298,13 @@
     - "Unity"
     - "Scratch"
   icon: "gausssense.png"
+-
+  name: "devRant.io"
```

Comment:
If you aren't affiliated with `devRant.io` I think you should add it to the `name` on line `1302` as well. Ready to merge after that 👍